src/stringBuffer.js

```diff
@@ -29,7 +29,19 @@
  * @return {function}
  */
 function makeBuffer() {
-  // write code here
+  let str = '';
+
+  function buffer(text) {
+    if (arguments.length === 0) {
```

Comment:
This is an implicit way to check `text` is not passed. You might as well check that text is undefined. That's more explicit and easier to read.